src/Extensions/Services.Remoting/Internal/Client/OmexServiceRemotingClientFactory.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Services.Client;
+using Microsoft.ServiceFabric.Services.Communication.Client;
+using Microsoft.ServiceFabric.Services.Remoting.V2;
+using Microsoft.ServiceFabric.Services.Remoting.V2.Client;
+
+namespace Microsoft.Omex.Extensions.Services.Remoting.Client
+{
+	/// <summary>
+	/// Factory for creating <see cref="IServiceRemotingClientFactory" />
+	/// </summary>
+	internal class OmexServiceRemotingClientFactory : IServiceRemotingClientFactory
+	{
+		private readonly IServiceRemotingClientFactory m_serviceRemotingClientFactory;
+
+		public event EventHandler<CommunicationClientEventArgs<IServiceRemotingClient>>? ClientConnected;
+
+		public event EventHandler<CommunicationClientEventArgs<IServiceRemotingClient>>? ClientDisconnected;
+
+		public OmexServiceRemotingClientFactory(IServiceRemotingClientFactory serviceRemotingClientFactory)
```

Comment:
This file probably needs descriptions for public items, although I'm not sure what the expectation is for internal classes.

Reply:
If calls itself is internal their public members won't be visible and don't require comments